src/Doctrine/Orm/Filter/DateFilter.php

```diff
@@ -269,4 +274,25 @@ protected function addWhere(QueryBuilder $queryBuilder, QueryNameGeneratorInterf
 
         $queryBuilder->setParameter($valueParameter, $value, $type);
     }
+
+    /**
+     * @return array<string, string>
+     */
+    public function getSchema(Parameter $parameter): array
+    {
+        return ['type' => 'date'];
+    }
+
+    public function getOpenApiParameters(Parameter $parameter): OpenApiParameter|array|null
+    {
+        $in = $parameter instanceof QueryParameter ? 'query' : 'header';
+        $key = $parameter->getKey();
+
+        return [
+            new OpenApiParameter(name: $key.'[after]', in: $in),
+            new OpenApiParameter(name: $key.'[before]', in: $in),
+            new OpenApiParameter(name: $key.'[strictly_after]', in: $in),
+            new OpenApiParameter(name: $key.'[strictly_before]', in: $in),
+        ];
+    }
 }
```

Comment:
note to self: we should probably log if a filter doesn't support the `:property` placeholder.